pkg/engine/update.go

```diff
@@ -142,10 +156,30 @@ func installPlugins(
 	return allPlugins, defaultProviderVersions, nil
 }
 
+func installAndLoadPolicyPlugins(plugctx *plugin.Context, policies []RequiredPolicy) error {
+	for _, policy := range policies {
```

Comment:
Seems like this ought to use the same sort of parallel implementation as `ensurePluginsAreInstalled`

Reply:
There's a comment about this. We will move it out later when there is a dedicated API call for getting a list of required policies. Right now it's in kind of an awkward place because this list is transmitted as part of creating the update.

Reply:
Sorry, how does that relate to doing these installations in parallel?

Reply:
Because using the actual plugin mechanism will allow us to install the plugins lazily and in parallel? Could be confused.

Reply:
I suppose as long as we're satisfied with the experience here for the MVP, then this is fine